docs/UserGuide.md

```diff
@@ -66,16 +66,28 @@ Adds a person to the address book.
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
 
 
-Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
+>**Examples**:
+>* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01`
+>* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 t/criminal`
 
 ### Listing all persons : `list`
 
 Shows a list of all persons in the address book.
 
 Format: `list`
 
+### Search for passengers: `search`
+
+Search passengers according to the constraints such as looking for female driver only or if they are available
+
+Format: `search CONSTRAINT`
+
+- Only one criteria can be search at a time
+- List of constraints that can be searched includes:
+  *availability*, *gender*
+
+> **Examples**:`search female` or `search available`
```

Comment:
Just this `> **Examples**` has an extra space compared to every other ones in markdown, but visual wise does not change anything